shared/src/metabase/types.cljc

```diff
@@ -106,18 +106,6 @@
 ;; TIMEZONE FIXME — not 100% sure this distinction is needed or makes sense.
 (derive :type/Instant :type/DateTimeWithLocalTZ)
 
-(derive :type/UNIXTimestamp :type/Instant)
-(derive :type/UNIXTimestamp :type/Integer)
-(derive :type/UNIXTimestampSeconds :type/UNIXTimestamp)
-(derive :type/UNIXTimestampMilliseconds :type/UNIXTimestamp)
-(derive :type/UNIXTimestampMicroseconds :type/UNIXTimestamp)
-
-(derive :type/TemporalString :type/Text)
-(derive :type/TemporalString :type/DateTime)
-(derive :type/ISO8601DateTimeString :type/TemporalString)
-(derive :type/ISO8601DateString :type/TemporalString)
-(derive :type/ISO8601TimeString :type/TemporalString)
-
```

Comment:
due to the weirdness of the migrations i'm thinking about leaving these in the type hierarchy for a bit but not exposing them to the frontend version of the type hierarchy. Meaning the backend wouldn't blow up and the frontend wouldn't be able to select them as a new choice.